src/state/api.js

```diff
@@ -13,6 +13,10 @@ export async function getAppIdData(appName, appId) {
   return fetchJSON(`/data/${appName}/app_ids/${appId}.json`);
 }
 
+export async function getLabelData(appName, labelName) {
```

Comment:
Why do we need this function? I don't see it being used anywhere?

Reply:
I think it's a leftover from an earlier iteration, will remove.